pkg/platformconfig/platformconfig.go

```diff
@@ -35,6 +36,7 @@ type Config struct {
 	CronTriggerCreationMode  CronTriggerCreationMode  `json:"cronTriggerCreationMode,omitempty"`
 	FunctionAugmentedConfigs []LabelSelectorAndConfig `json:"functionAugmentedConfigs,omitempty"`
 	IngressConfig            IngressConfig            `json:"ingressConfig,omitempty"`
+	KubeDefaultServiceType   v1.ServiceType           `json:"kubeDefaultServiceType,omitempty"`
```

Comment:
Add Kube block and inside it defaultServiceType.
Add TODO to move IngressConfig inside that block in a BC manner